base/musl/Dockerfile

```diff
@@ -51,6 +51,8 @@ ARG RECCASTER_VERSION
 ARG IPAC_VERSION
 ARG CAPUTLOG_VERSION
 ARG RETOOLS_VERSION
+ARG OPCUA_VERSION
```

Comment:
We don't have the static library for OPC-UA as far as I know. That means musl image won't be able to use build it anyway. Thus, we may remove this definition here and from its `docker-compose.yml`.

@ericonr, please correct me if I'm mistaken about this.